docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -103,20 +137,147 @@ To list all available releases, you can use the following command from the root
+
+You can list all the options for `download-results.py` by running the following from the root directory of the repository:
+
+```sh
+./download-results.py --help
+```
+
+!!! tip "Log into AWS CLI before running the script"
+    Before running this script locally, you will need to be [logged into your AWS CLI profile](../../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session).
+    To do this, run the following commands in terminal before running the `results-download.py` script, and follow instructions to log in.
+
+    ```sh
+    # replace `openscpca` with your AWS CLI profile name if it differs
+    export AWS_PROFILE=openscpca
+    aws sso login
+    ```
+
+    The command `export AWS_PROFILE=openscpca` will define your AWS profile name as `openscpca` for the duration of your terminal session.
+    Defining this variable is helpful because the `download-data.py` script also needs your profile name to download data from S3.
+
+    Note that you can also [add this profile definition to your shell profile file](../../technical-setup/environment-setup/configure-aws-cli.md#storing-your-aws-profile-name) so that it will always be defined in any terminal session.
+Example script usage below assumes you have created a profile called `openscpca` when [configuring the AWS CLI](../../technical-setup/environment-setup/configure-aws-cli.md).
+
+To download all results for one or more modules, use the `--modules` option as follows:
+
+```sh
+# Get results from a single module
+./download-results.py --modules module-name
+
+# Get results from several modules
+./download-results.py --modules first-module,second-module
+```
+
+To list all available modules for which you can download results, you can use the `--list-modules` option as follows.
+This will list all modules available in, by default, the current release:
+
+```sh
+./download-results.py --list-modules
+```
+
+To review what result files would be downloaded without performing the download yet, you can use the `--dryrun` option as follows:
+
+```sh
+./download-results.py --modules module-name --dryrun
+```
+
+While the structure of results files varies by module, some modules' results will be organized by project and/or sample.
+For those modules, you can use either the `--projects` or `--samples` flag (noting that these options are mutually exclusive) to download results specific to one or more projects or samples:
+
+```sh
+# use the --projects option
+./download-results.py --modules module-name --dryrun --projects SCPCPXXXXXX
+
+# use the --samples option
+./download-results.py --modules module-name --dryrun --samples SCPCSXXXXXX,SCPCSXXXXXY
+```
+
+
+#### Download result data structure
+
+By default, results will download into the `data` directory in your local copy of the `OpenScPCA-analysis` repository, but this can be customized with the `--data-dir` option.
```

Comment:
I don't know if we want to call out this customization... just that using a different location will likely break scripts.